File: codechef/medium/CLGOPJP326/solution.java
import java.util.Scanner;

class Book {
    int code;
    String title;
    
    double price;

    // Constructor to initialize
    public Book(int code, String title, double price) {
        this.code = code;
        this.title = title;
        this.price = price;
    }

    // Display method
    public void display() {
        System.out.println("Code: " + code);
        System.out.println("Title: " + title);
        System.out.printf("Price: %.2f\n" , price);
    }
}

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int code = sc.nextInt();
        String title = sc.next();
        double price = sc.nextDouble();

        Book book = new Book(code,title,price);
        
        book.display();
        
    }
}

File: codechef/medium/CLGOPJP326/solution.cpp
import java.util.Scanner;

class EmployeeSalary {
    // Method to calculate the gross salary with return type
    public static double calculateSalary(double basicSalary , double HRA, double DA) {
        return basicSalary + HRA + DA;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Taking input from the user for basic salary
        double basicSalary = scanner.nextDouble();
        
        // Calculate HRA and DA
        double HRA = 0.10 * basicSalary;
        double DA = 0.05 * basicSalary;
        
        double grossSalary = EmployeeSalary.calculateSalary(basicSalary, HRA , DA);
        // Displaying the details
        System.out.println("Employee Salary Details:");
        System.out.println("Basic Salary: " + basicSalary);
        System.out.println("HRA: " + HRA);
        System.out.println("DA: " + DA);
        System.out.println("Gross Salary: " + grossSalary);
        
        scanner.close();
    }
}
